test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Error.Tests.ps1

```diff
@@ -107,23 +107,15 @@ Describe 'Get-Error tests' -Tag CI {
     }
 
     It 'Get-Error adds ExceptionType for Exceptions' {
+
         try {
-            [System.Net.DNS]::GetHostByName((New-Guid))
+            throw [ArgumentException] 'myexception'
         }
         catch {
         }
 
         $out = Get-Error | Out-String
-        $out | Should -BeLikeExactly '*Type*'
-
-        if ($IsWindows) {
-            $expectedExceptionType = "System.Management.Automation.ParentContainsErrorRecordException"
-        }
-        else {
-            $expectedExceptionType = "System.Net.Internals.SocketExceptionFactory+ExtendedSocketException"
-        }
-
-        $out | Should -BeLikeExactly "*$expectedExceptionType*"
+        $out | Should -Match "Type\s+:\s(.\[\d+m)?System.ArgumentException"
```

Comment:
Why don't we follow previous test and don't check the added property directly?

```c#
        $null = Get-Error
        $error[0].Exception.Type | Should -BeExactly "System.ArgumentException"
```

Reply:
Here we are testing the formatting, so we need to check exactly how the string looks.